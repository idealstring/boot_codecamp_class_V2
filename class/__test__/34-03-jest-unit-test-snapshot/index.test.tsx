import { render } from "@testing-library/react";
import JestUnitTestPage from "../../pages/34-03-jset-unit-test-snapshot";

it("기존 사진이랑 바뀐게 없는 지 비교해보자! - 스냅샷 테스트", () => {
  const result = render(<JestUnitTestPage />);
  expect(result.container).toMatchSnapshot();
  // container 렌더링한 결과
  // toMatchSnapshot 기존 사진과 비교하겠다. 기존 사진이 없으면 먼저 찍음. 있을 때 비교.
});
