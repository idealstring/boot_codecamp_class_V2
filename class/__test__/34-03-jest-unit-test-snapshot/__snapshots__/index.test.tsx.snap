// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진이랑 바뀐게 없는 지 비교해보자! - 스냅샷 테스트 1`] = `
<div>
  <div>
    <div>
      철수는 13살 입니다.
    </div>
    철수 취미 입력하기 : 
    <input
      type="text"
    />
    <button>
      철수랑 놀러가기
    </button>
  </div>
</div>
`;
